src/components/summary/_macro.njk

```diff
@@ -35,15 +35,15 @@
                                 {% if row.error is defined and row.error %} {% set itemClass = " ons-summary__item--error" %}{% endif %}
                                 {% if row.total is defined and row.total %} {% set itemClass = itemClass + " ons-summary__item--total" %}{% endif %}
 
-                                <tbody class="ons-summary__item{{ itemClass }}">
+                                <tbody {% if row.rowId is defined and row.rowId %}id="{{ row.rowId }}" {% endif %}class="ons-summary__item{{ itemClass }}">
                                     {% if row.errorMessage is defined and row.errorMessage or (row.rowItems | length > 1 and row.rowTitle) %}
                                         <tr class="ons-summary__row">
                                             <th colspan="3" class="ons-summary__row-title ons-u-fs-r">{{ row.errorMessage or row.rowTitle }}</th>
                                         </tr>
                                     {% endif %}
 
                                     {% for rowItem in row.rowItems %}
-                                        <tr class="ons-summary__row{{ " ons-summary__row--has-values" if rowItem.valueList else "" }}">
+                                        <tr {% if row.rowItemId is defined and row.rowItemId %}id="{{ row.rowItemId }}" {% endif %}class="ons-summary__row{{ " ons-summary__row--has-values" if rowItem.valueList else "" }}">
```

Comment:
As below, this should come from the row item itself, so like `rowItem.rowItemId`. Also given these are always in the context of what they are used in, do we need to prefix each thing?.

Maybe it should be:
1. `group.groupHeaderId` > `group.id`
2. `row.rowId` > `row.id`
3. `rowItem.rowItemId` > `rowItem.id`

Thoughts?

Reply:
Yeah I can see that, I don't really mind either way. I just thought it was more explicit with all the different ids that are now able to be set and there are already other params that have similar prefixes. @jrbarnes9 @boxadesign what do you think?

Reply:
I think it is better to just use the name `id`. It's provided inside it's context so shouldn't lead to any confusion. Also, if the parent name is changed it saves needing to pick through any other params that also contain the parent name as a prefix

Reply:
Yeah good point ill change it